### ISSUE TITLE
Multiple default values when using multiple: true

### ISSUE THREAD

**Issue by ohenrik** [brainspec/enumerize#186]
It should be possible to provide an array to the default attribute when the option multiple = true.

```
enumerize :interests, in: [:music, :sports], multiple: true, default: [:music, :sports]
```

today this throws an error.

**Comment by nashby** [comment]
@ohenrik hey, thank you for the idea! I've implemented it on the master branch please give it a try!

**Comment by ohenrik** [comment]
Awesome, no problem, thank you for implementing it! :)

The feature seems to be working good. However i had to use this github repo as source to get the feature, has this not been pushed to rubygems yet?

**Comment by nashby** [comment]
@ohenrik not yet. I'll let you know when it's released.

**Comment by ohenrik** [comment]
Perfect thank you!

**Comment by lest** [comment]
@ohenrik I released version 0.11.0. :sparkles:

**Comment by ohenrik** [comment]
Thank you! :tada: :cake:
